.github/workflows/validation.yml

```diff
@@ -18,6 +18,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2.3.4
+
+    - name: Create env file
+      run: |
+        touch .env
+        echo PASSPORT_API_KEY=${{ secrets.PASSPORT_API_KEY }}  >> .env
+        echo PASSPORT_SCORER_ID=${{ secrets.PASSPORT_SCORER_ID }}  >> .env
+
```

Comment:
@ChaituVR added this to the workflow so it can use the Passport keys you set as secret variables

Reply:
Yea already added them, but not sure why it doesn't read 🤔

Reply:
Works well in local

Reply:
tested locally using [https://github.com/nektos/act](act) and it's working. There might be an issue with how you're setting the secret env vars in the repo settings